swan-lake/integration-tutorials/sending-emails-from-a-service.md

```diff
@@ -99,158 +99,108 @@ Follow the instructions given in this section to develop the service.
     $ bal new sending-emails-from-a-service
     ```
 
-2. Introduce the source code in files with the `.bal` extension (e.g., the `main.bal` file). 
+2. Remove the generated content in the `main.bal` file and open the diagram view in VS Code.
 
-    Import the 
-    - `ballerina/email` module to send emails using the SMTP protocol
-    - `ballerina/http` module to develop the REST API and define the clients that can be used to send requests to the backend services
-    - `ballerina/log` module to log debug, error or info level information for each client request
+    ![Open diagram view](/learn/images/integration-tutorials/sending-emails-from-a-service/open_diagram_view.gif)
 
-    ```ballerina
-    import ballerina/email;
-    import ballerina/http;
-    import ballerina/log;
+3. Generate a record type corresponding to the payload from the backend service by providing a sample of the expected JSON payload.
```

Comment:
Since there's two let's make this plural.

```suggestion
3. Generate record types corresponding to the payloads from the backend services by providing samples of the expected JSON payloads.
```